pkg/console/service/service_test.go

```diff
@@ -147,7 +147,6 @@ var _ = Describe("Service", func() {
 		testService.TokenHandler(request, response)
 
 		Expect(recorder.Code).To(Equal(http.StatusUnauthorized))
-		Expect(recorder.Body.String()).To(Equal("authenticating token cannot be empty"))
```

Comment:
Do we not want to assert that the body is empty now?

Reply:
I've added the check.